reports/tests/test_serializers.py

```diff
@@ -0,0 +1,46 @@
+from reports.models import EnrolmentReport
+from reports.serializers import EnrolmentReportSerializer
+
+
+def test_enrolment_report_to_representation():
+    report = EnrolmentReport(
+        occurrence_languages=[["fi", "Finnish"]],
+        study_group_study_levels=[["age_0_2", "age 0-2"]],
+        keywords=[["kultus:5", "Musiikki"]],
+    )
+    serializer = EnrolmentReportSerializer(instance=report)
+    assert serializer.data["occurrence_languages"][0]["id"] == "fi"
+    assert serializer.data["occurrence_languages"][0]["name"] == "Finnish"
```

Comment:
```suggestion
    assert serializer.data["occurrence_languages"][0] == {"id": "fi", "name": "Finnish"}
```
would be more succinct and also verify that there is no extra data in the object.